astropy/time/tests/test_precision.py

```diff
@@ -310,7 +310,11 @@ def test_two_sum(i, f):
         assert_almost_equal(a, b, atol=Decimal(tiny), rtol=Decimal(0))
 
 
-@given(floats(), floats())
+# FIXME: This example fails, so need to maybe set bounds for test cases
+# but what are the correct bounds?
+#     @example(f1=-3.089785075544792e307, f2=1.7976931348623157e308)
+# See https://github.com/astropy/astropy/issues/12955#issuecomment-1186293703
+@given(floats(min_value=-1e99, max_value=1e99), floats(min_value=-1e99, max_value=1e99))
```

Comment:
Note: If we decide to have limits like this, maybe should do a limit check in `two_sum` (which will basically also set limits for `Time`) instead of hiding it in a test.

Reply:
I think it is OK to limit the values here; I guess logically, the limits should be a factor 2 below the maximum and minimum,  i.e., `2**1023` exclusive. Or, perhaps easier, `min_value=np.finfo('f8').min / 2., max_value=np.finto('f8').max / 2.`